arm9/overlays/24/include/overlay_24.h

```diff
@@ -0,0 +1,81 @@
+#ifndef POKEDIAMOND_OVERLAY_24_H
+#define POKEDIAMOND_OVERLAY_24_H
+
+typedef enum {
+    STYLUS_TYPE_ERASE = 0,
+    STYLUS_TYPE_DRAW = 1,
+} StylusType;
+
+typedef enum {
+    MEMO_PAD_PIXEL_TYPE_FILLED = 1,
+    MEMO_PAD_PIXEL_TYPE_EMPTY = 4,
+} MemoPadPixelType;
+
+typedef struct MemoPadDrawState MemoPadDrawState;
+struct MemoPadDrawState
+{
+    StylusType stylusType;
+    u32 lastModifiedX;
+    u32 lastModifiedY;
+    u8 pixelData[78][75];
+    u16 padding;
+    u32 unk16E8;
+};
+
+typedef struct MemoPadDisplayHandler MemoPadDisplayHandler;
+struct MemoPadDisplayHandler
+{
+    MemoPadDrawState* drawState;
+    BgConfig* config;
+    u32 unk8;
+    u8 unkC[0x44];
+    u32 unk50;
+    u8 unk54[0x14];
+    u32* unk68[2];
+    Window *window;
+};
+
+typedef struct MemoPadAppHandler MemoPadAppHandler;
+struct MemoPadAppHandler
+{
+    u8 unk0;
+    u8 unk1;
+    u8 unk2;
+    u8 unk3;
+    u32 unk4;
+    MemoPadDrawState drawState;
+    MemoPadDisplayHandler* displayHandler;
+    u32 unk16F8;
+    u32 unk16FC;
+};
+
+BOOL ov24_02254854(MemoPadAppHandler**, int, int, int);
```

Comment:
these should have parameter names, also please use s32 and not int here, pointer alignment also applies